scylla-cdc/src/customer.rs

```diff
@@ -0,0 +1,113 @@
+use std::collections::HashMap;
+use scylla::frame::response::result::{ColumnSpec, CqlValue, Row};
+
+pub trait Consumer {
+    fn consume_cdc(&self, data: CDCRow);
+}
```

Comment:
Two things:

- We want to allow consumers to keep some internal state an probably modify it. Therefore, `consume_cdc` should take `self` as a mutable reference.
- We want to allow the library to read streams from multiple threads, therefore we won't be sharing a single instance of `Consumer` for processing all streams - instead, each stream (or each group of streams) will get a separate `Consumer` instance. Because the library is responsible for handling stream IDs and not the user, it should have ability to create new `Consumers` itself. The point is that we need to define a `ConsumerFactory`, too.